features/step_definitions/page_steps.rb

```diff
@@ -126,6 +126,7 @@
 When /^I change the slug to "([^"]*)"/ do |path|
   fill_in 'Slug', :with => path
   click_button 'Update Page'
+  page.should_not have_content('Update Page')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.